packages/legacy/core/App/navigators/RootStack.tsx

```diff
@@ -46,8 +46,14 @@ const RootStack: React.FC = () => {
   const navigation = useNavigation<StackNavigationProp<AuthenticateStackParams>>()
   const theme = useTheme()
   const defaultStackOptions = createDefaultStackOptions(theme)
-  const { splash, showPreface, enableImplicitInvitations, enableReuseConnections, enableUseMultUseInvitation } =
-    useConfiguration()
+  const {
+    splash,
+    showPreface,
+    enableImplicitInvitations,
+    enableReuseConnections,
+    enableUseMultUseInvitation,
+    pushNotification,
```

Comment:
Should we rename this `enablePushNotifications` or something like that? What do you think

Reply:
ya I think that would be a good idea